src/Linters/AutoFixingLineLinter.php

```diff
@@ -31,4 +31,4 @@
     }
     \file_put_contents($this->getFile(), Str\join($lines, "\n"));
   }
-}
```

Comment:
is your editor doing this, or hackfmt? are you on windows?

Reply:
This was the hackfmt command from the other PR. I can manually revert those lines.